tests/PHPStan/Rules/Generators/YieldFromTypeRuleTest.php

```diff
@@ -39,7 +39,6 @@ public function testRule(): void
 			[
 				'Generator expects value type array{DateTime, DateTime, stdClass, DateTimeImmutable}, array{0: DateTime, 1: DateTime, 2: stdClass, 4: DateTimeImmutable} given.',
 				74,
-				'Array does not have offset 3.',
```

Comment:
Since it's decided by the early return about list we don't have the information anymore.
- Not sure if it's an ok loss or we need to keep this (and how)
- Not sure if we need an help "This must be a list"

Your call.